app-office/taskcoach/metadata.xml

```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<longdescription>
```

Comment:
Maintainer info should be first.

Reply:
Sorry about that. I put the elements in the order they were listed [in the devmanual under the pkgmetadata element](https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html), and assumed that the DTD or schema would complain if they were in the wrong order. Corrected in fixup commit 2d6e3b90ad121f89b7d4e93ca3e5e1499f2bc7cc which can be squashed prior to merge.

Reply:
It's fine, if you look at the examples there though, maintainer comes before longdescription. Its not so strict, but I dont know if some metadata-scrawling tools breaks due to it. We try to stay consistent.